Makefile

```diff
@@ -58,15 +58,15 @@ endef
 
 # Set ACC_TEST_NAME to name of test in cli
 testoneacc:
-	$(call test_acc_common_env_vars) $(call test_acc_basic_auth_env_vars) TF_ACC=1 go test ./internal/acctest/... -timeout 10m -run ${ACC_TEST_NAME} -v count=1
+	$(call test_acc_common_env_vars) $(call test_acc_basic_auth_env_vars) TF_ACC=1 go test ./internal/acctest/config/... -timeout 10m -run ${ACC_TEST_NAME} -v count=1
 
 testoneacccomplete: spincontainer testoneacc
 
 # Some tests can step on each other's toes so run those tests in single threaded mode. Run the rest in parallel
 testacc:
-	$(call test_acc_common_env_vars) $(call test_acc_basic_auth_env_vars) TF_ACC=1 go test `go list ./internal/acctest/config... | grep -v -e authenticationapi -e oauth/authserversettings/scopes -e oauth/openidconnect/policy` -timeout 10m -v -p 4; \
+	$(call test_acc_common_env_vars) $(call test_acc_basic_auth_env_vars) TF_ACC=1 go test `go list ./internal/acctest/config... | grep -v -e authenticationapi -e oauth/authserversettings/scopes -e oauth/openidconnect/policy -e oauth/openidconnect/settings` -timeout 10m -v -p 4; \
```

Comment:
Why does this test need to be in the second group?

Reply:
When I finish, I have to remove the server-profile provided policy `oidcPolicy` in order for the null-object test at the end to work.  Setting this to default in the tests attaches to this policy and my cleanup fails due to the dependency. 

Since I am removing this policy, other tests that depend on it would fail if it were not there, so putting it at the end made the most sense.